src/main.rs

```diff
@@ -252,9 +276,86 @@ async fn main() {
                 println!("Link data:\n{:?}", link.unwrap());
             }
         },
+
+        Command::File(file) => match &file {
```

Comment:
We also need a command to list storages in `.ark` folder.

For simplicity, we could just use path constants from `arklib` (`lib.rs`). We'll need to update this `list` command manually when new storages are added, but I don't think it's worth to add marker files or special folder name for atomics.

The `list` command should also display:
* current latest version number
* app id of the latest version
* information about forks if such exist